server/models/vote.js

```diff
@@ -0,0 +1,17 @@
+const mongoose = require('mongoose');
+const Schema = mongoose.Schema;
+
+const VoteSchema = new Schema({
+  userId: {
+    type: Schema.Types.ObjectId,
+    required: true,
+    ref: 'users',
+  },
+  pollId: {
+    type: Schema.Types.ObjectId,
+    required: true,
+    ref: 'polls',
+  },
```

Comment:
Shouldn't it save which image the vote went to?

Reply:
You're right. This will be useful for displaying who voted for what on the single poll show page. Good catch.